app/src/main/res/values/strings.xml

```diff
@@ -59,6 +59,9 @@
   <string name="settings_notifications_about_to_start_summary_on">Show notifications for favourite events starting soon</string>
   <string name="settings_notifications_about_to_start_summary_off">Do not show notifications for favourite events starting soon</string>
   <string name="settings_message_signed_out">You have been signed out.</string>
+  <string name="about_title">About</string>
+  <string name="settings_account_not_logged_in">Not logged in</string>
+  <string name="settings_header_not_signed_in">You’re not signed in</string>
```

Comment:
Ok, so I just checked with him: let's stick with `signed in` copy. Please do it here so that we don't forget.
Also, there is a behaviour update needed: `you’re not supposed to display account data in the Account category when not logged in anyway`